src/base/misc/fatfs.c

```diff
@@ -543,11 +548,12 @@ static char *system_type(unsigned int t) {
     case PC_D:
         return "Unknown PC-DOS";
     case DR_D:
-        return "DR-DOS";
-/*  case DRO_D:
-        return "Old DR-DOS"; // Duplicate case to PC_D at the moment
-*/
-    case REALPCD_D:
+        return "Old DR-DOS (< 5.0)";
+    case MIDDRD_D:
+        return "Original DR-DOS (>= v5.0 && <= v8.0) && OpenDOS (<= 7.01.06)";
```

Comment:
Yes, agreed. I'll wait before fixing it, just in case testing anything else drops out of testing.